python_example/util/vec.py

```diff
@@ -0,0 +1,85 @@
+import math
+
+
+# This is a helper class for vector math. You can extend it or delete if you want.
+class Vec3:
+    """
+    This class should provide you with all the basic vector operations that you need, but feel free to extend its
+    functionality when needed.
+    The vectors found in the GameTickPacket will be flatbuffer vectors. Cast them to Vec3 like this:
+    `car_location = Vec3(car.physics.location)`.
+
+    Remember that the in-game axis are left-handed.
+
+    When in doubt visit the wiki: https://github.com/RLBot/RLBot/wiki/Useful-Game-Values
+    """
+
+    def __init__(self, x: float=0, y: float=0, z: float=0):
+        if hasattr(x, 'x'):
+            # We have been given a vector. Copy it
+            self.x = float(x.x)
+            self.y = float(x.y) if hasattr(x, 'y') else 0
+            self.z = float(x.z) if hasattr(x, 'z') else 0
+        else:
+            self.x = float(x)
+            self.y = float(y)
+            self.z = float(z)
+
+    def __add__(self, other):
+        return Vec3(self.x + other.x, self.y + other.y, self.z + other.z)
+
+    def __sub__(self, other):
+        return Vec3(self.x - other.x, self.y - other.y, self.z - other.z)
+
+    def __neg__(self):
+        return Vec3(-self.x, -self.y, -self.z)
+
+    def __mul__(self, scale):
+        return Vec3(self.x * scale, self.y * scale, self.z * scale)
+
+    def __rmul__(self, scale):
+        return self * scale
+
+    def __truediv__(self, scale):
+        scale = 1 / float(scale)
+        return self * scale
+
+    def __str__(self):
+        return "Vec3(" + str(self.x) + ", " + str(self.y) + ", " + str(self.z) + ")"
+
+    def flat(self):
+        """Returns a new Vec3 that equals this Vec3 but projected onto the ground plane. I.e. where z=0."""
+        return Vec3(self.x, self.y, 0)
+
+    def length(self):
+        """Returns the length of the vector."""
```

Comment:
in the documentation can you also mention other terms that this covers (ex magnitude)